packages/postgres-datasource/index.ts

```diff
@@ -173,7 +186,7 @@ export class PostgresDataSource implements DBOSDataSource<PostgresTransactionOpt
 
   // eslint-disable-next-line @typescript-eslint/no-empty-object-type
   static async initializeInternalSchema(options: postgres.Options<{}> = {}): Promise<void> {
-    const pg = postgres({ ...options, onnotice: () => {} });
+    const pg = postgres({ ...options /*onnotice: () => {}*/ });
```

Comment:
Is this ` /*onnotice: () => {}*/ ` still necessary?

```suggestion
    const pg = postgres({ ...options });
```

Reply:
oof, no it's still needed. I was debugging something so I removed this to enable NOTICE message logging